metadata-ingestion/source_docs/bigquery.md

```diff
@@ -0,0 +1,111 @@
+# BigQuery
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[bigquery]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Metadata for databases, schemas, and tables
+- Column types associated with each table
+
+:::tip
+
+You can also get fine-grained usage statistics for BigQuery using the `bigquery-usage` source described below.
+
+:::
+
+## Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
```

Comment:
Where do we state where to put the configurations below? We need to add links to all of these saying.. New to writing ingestion recipes? See `Getting started with Metadata Ingestion` which basically covers: How ingestion works. What a recipe file is. And scheduling... thoughts?